src/main/scala/com/ocadotechnology/Endpoints.scala

```diff
@@ -1,42 +1,30 @@
 package com.ocadotechnology
 
-import sttp.tapir.*
 
-import Library.*
-import cats.effect.IO
-import io.circe.generic.auto.*
+import sttp.tapir.*
+import sttp.tapir.json.circe.{jsonBody, *}
 import sttp.tapir.generic.auto.*
-import sttp.tapir.json.circe.*
-import sttp.tapir.server.ServerEndpoint
-import sttp.tapir.swagger.bundle.SwaggerInterpreter
+import eu.timepit.refined.types.string.NonEmptyString
+import eu.timepit.refined.auto.*
+import io.circe.generic.auto.*
+import io.circe.refined.*
+import com.ocadotechnology.repositories.UserRepository
+import com.ocadotechnology.models.{User, UserView}
+import sttp.tapir.codec.refined.TapirCodecRefined
+
```

Comment:
```suggestion
import sttp.tapir.codec.refined.*
```
to solve the problems with refined :wink: